### ISSUE TITLE
Change shortcut key for "Tired of this song"

### ISSUE THREAD

**Issue by retrodaredevil** [pithos/pithos#625]
I don't use the shortcut keys on pithos very much because I'm not usually focused on the window. However, sometimes I will think I'm focused in my browser window, then I'll hit CTRL+T to open a new tab, only to realize that one of my favorite songs won't be played for the next 30 days because Pithos was focused.

It would really be awesome if we could have an option to change the shortcut keys or disable some of them entirely.

**Comment by TheKrafter** [comment]
What about an environment var that disables them? like if `PITHOS_DISABLE_SHORTCUTS=1` then we just won't have shortcuts? 

Seems like a simple solution to me and would be easier to implement than a whole shortcut editing system.

**Comment by retrodaredevil** [comment]
> What about an environment var that disables them? like if `PITHOS_DISABLE_SHORTCUTS=1` then we just won't have shortcuts?

I wouldn't be apposed to this, but this issue (at least for me) is very specific to the shortcut key for "Tired of this song" because there is no way to undo that action (unlike with thumb up or thumb down). Personally I would only want to disable the "Tired of this song" shortcut.

**Comment by TheKrafter** [comment]
This might be overthinking it, but maybe we could have a list of keywords (like `TIRED`, `BAN`, `LOVE`, etc.) that if they're in the `PITHOS_DISABLE_SHORTCUTS` env var then we don't load them? For example you could set `PITHOS_DISABLE_SHORTCUTS=TIRED` to disable Ctrl+T to label tired of track? (we could also do `ALL` to disable all shortcuts)

This might seem like over-engineering the problem but it seems weird to have this feature for just one shortcut.

**Comment by TheKrafter** [comment]
This is where these shortcuts are defined: https://github.com/pithos/pithos/blob/231052b3b8a082ba13e2637253fe396ba162a2b4/pithos/pithos.py#L420-L479

So a few if statements checking environment variables for each action should be fine, right?

**Comment by TingPing** [comment]
Environment variables are a bad solution... but a good solution is a lot of work.

Something like https://github.com/GNOME/gnome-builder/tree/795bccca24b64ffd0f51608f52716eb14b3f535a/src/plugins/shortcutui

**Comment by TheKrafter** [comment]
yeah, i feel like this is an edge-case that we shouldn't put *that* much time into. Environment variables are an easy `if not "TIRED" in os.getenv["PITHOS_DISABLE_SHORTCUTS"]:`.

**Comment by TheKrafter** [comment]
So here's my proposed idea:  https://github.com/pithos/pithos/compare/master...TheKrafter:pithos:master
Note that this does not remove them from the "Shortcuts" UI, because that's hardcoded in a `.ui` file, but users using this feature are more than likely to know that they've disabled such keybinds.
- [x] I have tested that this works well and as intended
